scripts/update_framework.py

```diff
@@ -0,0 +1,27 @@
+from notion.client import NotionClient
```

Comment:
This is very much an initial temporary approach for the MVP. I've started working a sketch of how we might want to manage EDF in future directly in a repo, rather than with Notion as the SSOT.

Reply:
Okay - I'm happy if we move most of these to comments, but could we get a `requirements.txt` at least, just so that I can be sure we're always running the same code with this.

Reply:
Great. I've pushed in requirements.txt and opened #34 with synthesis of the issues from here.